x/protocolpool/keeper/abci.go

```diff
@@ -0,0 +1,40 @@
+package keeper
+
+import (
+	"context"
+	"time"
+
+	"cosmossdk.io/x/protocolpool/types"
+
+	"github.com/cosmos/cosmos-sdk/telemetry"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func (k *Keeper) EndBlocker(ctx context.Context) error {
+	defer telemetry.ModuleMeasureSince(types.ModuleName, time.Now(), telemetry.MetricKeyEndBlocker)
+
+	logger := k.Logger(ctx)
+
+	// Iterate over all continuous fund proposals and perform continuous distribution
+	// Note:  This loop intentionally processes all payment streams, and the number of streams
+	// can impact block processing time. However, since it is governed by governance, it is not
```

Comment:
should we put a limit on how many to do per block here? The other thing we could which is cool, is that when one person claims in a lazy fashion it claims it for everyone else as well.

Reply:
If we put a limit here, there is a chance that some continuous fund proposals being processed more frequently than others and may introduce delays in processing funds for some recipients. Users might experience longer wait times then. That wouldn't be fair for everyone.